src/Javascript/fs.js

```diff
@@ -41,10 +41,13 @@ export async function add({ blobs, pathSegments }) {
   return await listDirectory({ pathSegments })
 }
 
-export async function write({ pathSegments, text }) {
+const encoder = new TextEncoder()
```

Comment:
Should we put this in `writeItemUtf8` just to be safe?
I don't understand why they made this a class 😅

Reply:
I wanted to only 'allocate' this class once for performance/memory reasons, but I guess that's premature optimization.

Yep. Super annoying that this has to be a class. Not sure whether it actually stores any state. It definitely shouldn't.